pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -233,27 +277,40 @@ spec:
       replicas: 1
       selector:
         matchLabels:
-          {{- if eq $isMonitor "true" }}
-          monitoring: volume_exporter_prometheus
-          {{- end}}
+          app: cstor-volume-manager
           openebs.io/target: cstor-target
           openebs.io/persistent-volume: {{ .Volume.owner }}
-          app: cstor-volume-manager
       template:
         metadata:
           labels:
             {{- if eq $isMonitor "true" }}
             monitoring: volume_exporter_prometheus
+            openebs_pv: {{ .Volume.owner }}
```

Comment:
Are we following `_` convention?

Reply:
Added for backward compability with prometheus config. When it gets changed to just use openebs.io/persistent-volume, we can remove this.